### ISSUE TITLE
nvda doesn't announce the check boxes status in some cases

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#2165]
**Reported by fatma.mehanna on 2012-03-14 13:35**
hi,
sometimes nvda doesn't report the check box status, i.e if it is checked or not checked.
for example, go to skype>options>sounds.
try to check or uncheck one of its settings. nvda won't report if this check box is selected or not when you press space,
but it will report that when you press space and down arrow then highlight the option again or when you press nvda+tab.

**Comment by ehollig** [comment]
Using Skype 7.38.0.101, NVDA announces the check box status in the sounds settings options. IE. Checked or not checked. Looking at Skype releases, it appears that there has been many releases and changes since Skype 7.0. Closing
